src/gui/gui_msgs.h

```diff
@@ -35,50 +35,52 @@ There is NO WARRANTY, to the extent permitted by law.
 
-  -c <command>        Runs the specified DOS command before running FILE.
-                      Multiple commands can be specified.
+  -noautoexec          Don't perform any [autoexec] actions.
 
-  -noautoexec         Don't perform any [autoexec] actions.
+  -noprimaryconf       Don't read settings from the primary configuration file
+                       located in your user folder.
 
-  -noprimaryconf      Don't read settings from the primary configuration file
-                      located in ~/.config/dosbox/dosbox-staging.conf.
+  -nolocalconf         Don't read settings from "dosbox.conf" if present in
+                       the current working directory.
 
-  -nolocalconf        Don't read settings from "dosbox.conf" if present in
-                      the local current working directory.
+  -conf <configfile>   Start DOSBox with the options specified in <configfile>.
+                       Multiple configfiles can be specified.
 
-  -conf <configfile>  Start dosbox with the options specified in <configfile>.
-                      Multiple configfiles can be present at the commandline.
+  --working-dir <path> Set working directory to <path>. DOSBox will act as if
+                       started from this directory.
 
-  -fullscreen         Start dosbox in fullscreen mode.
+  -fullscreen          Start DOSBox in fullscreen mode.
 
-  -lang <langfile>    Start dosbox with the language specified in
-                      <langfile>.
+  -lang <langfile>     Start DOSBox with the language specified in <langfile>.
 
-  --list-glshaders    List available GLSL shaders and their directories.
-                      Results are useable in the "glshader = " conf setting.
+  --list-glshaders     List available GLSL shaders and their directories.
+                       Results are useable in the "glshader = " config setting.
 
-  -machine <type>     Setup dosbox to emulate a specific type of machine.
-                      The machine type has influence on both the videocard
-                      and the emulated soundcards.  Valid choices are:
-                      hercules, cga, cga_mono, tandy, pcjr, ega, vgaonly,
-                      svga_s3 (default), svga_et3000, svga_et4000,
-                      svga_paradise, vesa_nolfb, vesa_oldvbe.
+  -machine <type>      Setup DOSBox to emulate a specific type of machine.
+                       The machine type has influence on both the videocard
+                       and the emulated soundcards. Valid choices are:
+                       hercules, cga, cga_mono, tandy, pcjr, ega, vgaonly,
+                       svga_s3 (default), svga_et3000, svga_et4000,
+                       svga_paradise, vesa_nolfb, vesa_oldvbe.
 
-  -exit               Dosbox will close itself when the DOS program
-                      specified by FILE ends.
+  -exit                DOSBox will exit after the DOS program specified by
+                       FILE has ended.
```

Comment:
Can `DOSBox` be removed from the above descriptions?

Reply:
Yeah, good idea. I've simplified the descriptions even further.